README.md

```diff
@@ -89,20 +89,35 @@ Development
 rake
 ```
 
-A few [Cucumber][cucumber] features provide an end-to-end test harness, and a set of [RSpec][rspec] specs provide both a more granular documentation and a unit test harness.
-
-RSpec [tags][tags] are used to categorize the spec examples.
+A set of [RSpec][rspec] specs provide both a granular documentation and a unit test harness. RSpec [tags][tags] are used to categorize the spec examples.
 
 Spec examples that are tagged as `public` describe aspects of the gem public API, and MAY be considered as its documentation.
 
 The `private` or `protected` specs are written for development purpose only. Because they describe internal behaviour which may change at any moment without notice, they are only executed as a secondary task by the [continuous integration service][travis] and SHOULD be ignored.
 
 Run `rake spec:public` to print the gem public documentation.
 
-  [cucumber]: https://github.com/cucumber/cucumber-rails
+#### Behavioral Spec
+
+A few [Cucumber][cucumber] features from the [dredd-hooks-template] provide an end-to-end test harness. The test suite is provided as a [Node.js] package and needs to be installed by `npm`:
+
+```
+npm install
+```
+
+Then the tests can be ran using the `npm test` command, but for the Ruby paths to be correctly set, an environment variable `BUNDLE_GEMFILE` needs to be set to the path to the `Gemfile`. An example how running the end-to-end tests can look like on macOS:
+
+```
+BUNDLE_GEMFILE=$(pwd)/Gemfile npm test
```

Comment:
I was able to run the test suite locally following these installation and execution instructions. :heavy_check_mark: